.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{*.json,*.yaml,*.yml}]
```

Comment:
@pshoniuk did you want to commit this? not sure if these format rules conflict with prettier?

Reply:
This configuration doesn't conflict with prettier rules. EditorConfig helps maintain common settings between different IDEs. For example, I use WebStorm, and my default setting defines two spaces for yaml files, but this project uses 4